packages/shared-ui/src/components/header/Transition.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { Fragment } from 'react'
```

Comment:
This component needs to be moved to common UI from header once it's moved to shared-ui (probably)